### ISSUE TITLE
Swagger Spec differs from actual Sync Gateway response

### ISSUE THREAD

**Issue by curiousily** [couchbaselabs/couchbase-mobile-portal#621]
Hi,

We're trying to create Swagger client using this spec:

https://s3.amazonaws.com/cb-docs-swagger/specs/sync-gateway-public.json

When we hit our Sync Gateway (1.4) instance with POST /{db}/ the status code of the response is 200. Within the spec the response code is defined as 201:

https://github.com/couchbaselabs/couchbase-mobile-portal/blob/master/swagger/14/paths/cbl.yaml#L144

Are there other inconsistencies? Is there a problem with our understanding?

What is the recommended way of obtaining the spec file?

**Comment by jamiltz** [comment]
The first link in your comment is the spec when we first published it but then we realised we should probably version it:

- https://couchbase-docs.s3.amazonaws.com/mobile/1.3/rest-apis/sync-gateway-public.json
- https://couchbase-docs.s3.amazonaws.com/mobile/1.4/rest-apis/sync-gateway-public.json

Whilst the link is updated on the master branch of this repo (`swagger/14` and `swagger/13`) we haven't updated the guides in the documentation to refer to the versioned link. And also put in place the necessary redirects. Keeping this ticket open to track this. Let me know if using the 1.4 link resolves your issue.
